packages/frontend/amp/components/TopMetaPaidContent.tsx

```diff
@@ -53,8 +64,8 @@ const PaidForByLogo: React.FC<{
             >
                 <amp-img
                     src={logo.src}
-                    width={`${logo.dimensions.width}px`}
-                    height={`${logo.dimensions.height}px`}
+                    width={`140px`}
```

Comment:
Does the logo have restricted dimensions up stream?

Reply:
So I spoke to @paperboyo about this. Most logos coming in are 280x180 but some are smaller at 140x90. They increased the src image dimensions so that logos would render less blurry.  The worry using a variable here is that if a very large logo image is uploaded by mistake it will render horribly so went back to hardcoding the smaller size. Looks better than the larger logos design-wise too.